### PR TITLE
Bug/post show

### PR DESCRIPTION
**Pull request by ksowndar**
when user is not signed in the post show page shows error it is now corrected

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -41,62 +43,74 @@
                   method: :delete,
                   data: { confirm: "Are you sure?" } %>
         </div>
-        <% if ((@post.account_id == current_account.id) and !(@post.closed?)) or !(@post.closed?) %>
+      <% end %>
+         <% if ((@post.account_id == current_account.id) and !(@post.closed?)) or !(@post.closed?) %>
```
Comment (iMagesh) [inline app/views/posts/show.html.erb:47]: why have you used 'and' and 'or' for @post.closed?

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -41,62 +43,74 @@
...
+        <% end %> 
         <button type="button" class="join-btn" data-toggle="modal" data-target="#myModal" remote = "true">
           Report
        </button>
       </div>
 	</div>  
-<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
-  <div class="modal-dialog" role="document">
-    <div class="modal-content">
+   <div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```
Comment (iMagesh) [inline app/views/posts/show.html.erb:58]: Can you move the modal to a separate partial?

Comment (ksowndar) [reply app/views/posts/show.html.erb]: Yes

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (ksowndar) [reply app/views/posts/show.html.erb]: It is changed

**Pull request merged**